mlflow/keras.py

```diff
@@ -27,14 +27,14 @@
 FLAVOR_NAME = "keras"
 
 DEFAULT_CONDA_ENV = _mlflow_conda_env(
-    additional_conda_deps=[
```

Comment:
There is a difference in tensorflow from pip / conda.  I think the conda tensorflow is supposed be built with mkl support and is support to be better on a cpu.